src/apiFunctions.js

```diff
@@ -0,0 +1,13 @@
+const getPokemon = () => fetch('https://pokeapi.co/api/v2/pokemon/?limit=20&offset=0/').then((response) => response.json());
+
+const arrayPokemonLinks = async () => {
+  const data = await getPokemon();
+  const array = data.results;
+  const urlArray = [];
+  array.forEach((element) => {
+    urlArray.push(element.url);
+  });
```

Comment:
https://github.com/MacCrazyman/pokemon-capstone/blob/fd0a43e3663ce80be1c3a303f1535e76eb4f6886/src/index.js#L31-L38

Please take a look at those `forEach` loops. It is possible that you are repeating yourself.
Check if there is any way to refactor those loops to keep only one, otherwise, let me know.